src/common/icons/index.js

```diff
@@ -79,5 +94,20 @@ export {
   gridIcon,
   boxIcon,
   databaseIcon,
-  phoneIcon
+  phoneIcon,
+  doctorDashboardIllustrator
+  // microscopeIcon,
+  // weightLifterIcon,
+  // fileSearchIcon,
+  // hairIcon,
+  // familyTreeIcon,
```

Comment:
Delete these comments too.

Reply:
i have deleted this comments @agustinustheo